sessions-form.html

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
@dcwalk should the filename be `session-confirmation`? Looking at the URL at https://github.com/ournetworks/2019/pull/66/files#diff-05c96faad7724d9a3e068b6976a623ddR14